### PR TITLE
feat: new active rounds section in secondary navbar

### LINKED ISSUE #2275: Explorer: update round dropdown menu to showcase upcoming rounds
### Overview
We have some upcoming rounds on Grants Stack across new chains (Fantom, Arbitrum, Avalanche, zkSync, Polygon) that we should showcase on Explorer via the dropdown menu we currently have for GG18. Details below! This is a temporary manual solution while we are redesigning the Explorer Homepage. 

i.e. would look something like this: 
IMG_3155

designs here - [figma](https://www.figma.com/file/4J1WWx7cOd0uUNbI2uA4z0/Explorer-GG18?type=design&node-id=817-26547&mode=design&t=lIzhW2tM3OoEnymj-4)

### Tasks

- [x] Replace round links in the GG18 menu with links to upcoming rounds (use same font / layout), separated by chain (see below for live Fantom and Arbitrum rounds) 
- [x] Update GG18 brand asset to "Active Rounds" asset (coming soon) 

### Links

**Arbitrum** 
- The Gaming Round: https://explorer.gitcoin.co/#/round/42161/0xd0a086c37fbd20c44f3ba2cff69208d1b62f54e3 

- New Protocol Ideas Round: https://explorer.gitcoin.co/#/round/42161/0x8b70206844630d8c0a2a545e92d3c8d46a3ceaad

- Developer Tooling on Nova Round:https://explorer.gitcoin.co/#/round/42161/0x1d16f0eedf8ced25f288056ddcbb653d0f0451ad

- Education, Community Growth & Events Round: https://explorer.gitcoin.co/#/round/42161/0x59d79b22595b17af659ce9b03907615f53742c57

**Fantom**
- Vault Round 1: https://explorer.gitcoin.co/#/round/250/0x8dce7a66e0c310f9f89e847dba83b2344d589161 
 
**Details**
Say more

### PR DESCRIPTION
**Pull request by vacekj**
fixes #2275

### CODE REVIEW

**Review by boudra — DISMISSED** [review]
remember to use `git mv` to rename or move files, it's really difficult to see the changes

**Review by bhargavaparoksham — APPROVED** [review]

**Comment by melissa-neira** [comment]
@vacekj just want to call out that the button's background should be white and not gray, otherwise lgtm!

**Comment by michellema1208** [comment]
+1 to Melissa's comment - otherwise lgtm as well!

**Comment by vacekj** [comment]
@melissa-neira should the bg be white only when the dropdown is active? otherwise, according to figma, the bg of the button is `background: var(--gitcoin-grey-50, #F3F3F5);`

**Comment by melissa-neira** [comment]
@vacekj the bg should always be white, my bad - updated the figma

**Comment by vacekj** [comment]
@melissa-neira fixed

**Review by melissa-neira — APPROVED** [review]

**Review by bhargavaparoksham — APPROVED** [review]

File: packages/grant-explorer/src/features/discovery/RoundsSubNav.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                      >
+                        <img
+                          src={(() => {
+                            switch (round.chainId) {
+                              case ChainId.ARBITRUM:
+                                return "/logos/arb-logo.svg";
+                              case ChainId.FANTOM_MAINNET_CHAIN_ID:
+                                return "/logos/fantom-logo.svg";
+                            }
+                          })()}
```
Comment (bard) [inline packages/grant-explorer/src/features/discovery/RoundsSubNav.tsx:107]: If you use the ternary here, you avoid the odd-looking inline call and force yourself to make all outcomes explicit:

```ts
src={
  round.chainId === ChainId.ARBITRUM
      ? "/logos/arb-logo.svg"
      : round.chainId === ChainId.FANTOM_MAINNET_CHAIN_ID
      ? "/logos/fantom-logo.svg"
      : null;
}
```

File: packages/grant-explorer/src/features/discovery/RoundsSubNav.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  },
+  {
+    id: "0x8dce7a66e0c310f9f89e847dba83b2344d589161",
+    name: "Vault Round 1",
+    chainId: ChainId.FANTOM_MAINNET_CHAIN_ID,
+  },
+];
+
+export function RoundsSubNav(props: Props) {
+  const [open, setOpen] = useState(false);
```
[the remark this answers is not in the document]
Comment (vacekj) [reply packages/grant-explorer/src/features/discovery/RoundsSubNav.tsx:46]: true, will fix.

File: packages/grant-explorer/src/features/discovery/RoundsSubNav.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+export type SubNavRound = Pick<Round, "id"> & {
+  chainId: ChainId;
+  name: string;
+};
+
+type Props = {
+  onClick: (round: SubNavRound) => void;
+};
+
+const rounds: SubNavRound[] = [
```
[the remark this answers is not in the document]
Comment (vacekj) [reply packages/grant-explorer/src/features/discovery/RoundsSubNav.tsx:17]: This is a one-time change, otherwise we would likely store it somewhere else.

**Pull request merged**